components/datetime/src/pattern/item/ule.rs

```diff
@@ -58,6 +58,10 @@ use zerovec::ule::{AsULE, ULE};
 pub struct PatternItemULE([u8; 3]);
 
 impl PatternItemULE {
+    /// Given the first byte of the three-byte array that `PatternItemULE` encodes,
+    /// the method determins whether the discriminant in
+    /// the byte indicates that the array encodes the `PatternItem::Field`
+    /// or `PatternItem::Literal` variant of the `PatternItem`.
```

Comment:
perhaps add a separate line saying "Returns true when it is a `PatternItem::Field`"